test/unit/sumMultipleFinite.spec.ts

```diff
@@ -0,0 +1,71 @@
+import sumMultipleFinite from '../../src/sumMultipleFinite'
+import { expect } from 'chai'
+
+describe('sumMultipleFinite', function () {
+  it('should sum correctly one value', function () {
+    const object = {
+      a: 1
+    }
+    const result = sumMultipleFinite(object, ['a'])
+    expect(result).to.equal(1)
+  })
+
+  it('should sum correctly all existing values', function () {
```

Comment:
This is actually checking that is should only sum those value requested even though all values in the collection are finite, something like:

```
should sum only requested values when all values are finite
```